lib/gitlab/client/users.rb

```diff
@@ -123,6 +123,20 @@ def session(email, password)
       post('/session', body: { email: email, password: password }, unauthenticated: true)
     end
 
+    # Gets a list of user activities (for admin access only).
+    #
+    # @example
+    #   Gitlab.activities
+    #
+    # @param  [Hash] options A customizable set of options.
+    # @option options [Integer] :page The page number.
+    # @option options [Integer] :per_page The number of results per page.
+    # @option options [DateTime] :from The start date for paginated results.
```

Comment:
```suggestion
    # @option options [String] :from The start date for paginated results.
```

>Date string in the format YEAR-MONTH-DAY. For example, 2016-03-11. Defaults to 6 months ago.

https://docs.gitlab.com/ce/api/users.html#get-user-activities-admin-only

Reply:
Fixed